HPXMLtoOpenStudio/resources/airflow.rb

```diff
@@ -288,18 +288,30 @@ def self.get_values_from_air_infiltration_measurements(hpxml_bldg, cfa, weather)
 
   # Calculate ACH50 for annual energy simulation when only leakiness description is provided
   #
-  # @param hpxml_bldg [HPXML::Building] HPXML Building object representing an individual dwelling unit
-  # @return [Float] Calculated ACH50 value based on IECC zone, cfa, infiltration height, year built, foundation type, ducts and leakiness description
-  def self.calc_ach50(cfa, ncfl_ag, year_built, ceil_height, infil_volume, iecc_cz, foundations, ducts, leakiness_description)
+  # @param cfa [Double] Conditioned floor area in the dwelling unit (ft2)
+  # @param ncfl_ag [Double] Number of conditioned floors above grade
+  # @param year_built [Integer] Year the dwelling unit is built
+  # @param ceil_height [Double] Average ceiling height (ft2)
+  # @param infil_volume [Double] Infiltration volume (ft2)
+  # @param iecc_cz [String] IECC climate zone
+  # @param foundations [Array<Array>] Array of [fraction, foundation_type] pair for each foundation
+  # @param ducts [Array<Array>] Array of [fraction, duct_location] pair for each duct
+  # @param leakiness_description [String] Leakiness description to qualitatively describe the dwelling unit infiltration
+  # @param air_sealed [Boolean] True if the dwelling unit is air sealed
+  # @return [Double] Calculated ACH50 value based on IECC zone, cfa, infiltration height, year built, foundation type, ducts and leakiness description
+  def self.calc_ach50(cfa, ncfl_ag, year_built, ceil_height, infil_volume, iecc_cz, foundations, ducts, leakiness_description = nil, is_sealed = false)
```

Comment:
I'm confused, why do you allow for leakiness_description=nil? Don't we only need to call this method if it's not nil? If it's nil, then we should already have the quantitative leakage value.

Reply:
Yes that's right for our workflow, but for current HEScore it uses air_sealed instead of leakiness_description, we should only have one being passed between these two.

Reply:
Ah, right. Now that you say that, I realize we don't actually need the `air_sealed` input (sorry!) because HEScore uses [a mapping](https://hescore-hpxml.readthedocs.io/en/latest/translation/about.html#air-sealing-present) between `leakiness_description` and `air_sealed`. So, for HEScore, if air_sealed=true, we could pass in tight, and if it's false, we could pass in average. That would keep the method simple.

Reply:
Oh, you're right. Okay, in that case, let's just keep it how you have it. When we go to use this in HEScore, we can decide if we want to simplify/change things or not at that point.